app/src/pages/admin/projectEventsPage/eventsGrid/eventsGrid.scss

```diff
@@ -1,55 +1,50 @@
-.time-col {
-  width: 10%;
-  height: 70px;
-  vertical-align: top;
-  font-size: 12px;
-}
-.user-col {
-  width: 10%;
-  vertical-align: top;
-  font-size: 13px;
-  font-family: $FONT-SEMIBOLD;
-  @media (max-width: $SCREEN_XS_MAX) {
-    overflow: hidden;
-    text-overflow: ellipsis;
-  }
-}
-.action-col {
-  width: 20%;
-  vertical-align: top;
-  font-size: 13px;
-  text-transform: lowercase;
-}
-.object-type-col {
-  width: 15%;
-  vertical-align: top;
-  font-size: 13px;
-  font-family: $FONT-SEMIBOLD;
-  @media (max-width: $SCREEN_XS_MAX) {
-    overflow: hidden;
-    text-overflow: ellipsis;
-  }
-}
-.action-col:first-letter,

```

Comment:
Actually I removed only uppercase for the first letter (because now I use common translation for all values: in grid and in filter list).